x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -82,6 +101,38 @@ export class TelemetryEventsSender {
     }
   }
 
+  public async fetchDiagnosticAlerts() {
+    const query = {
+      expand_wildcards: 'open,hidden',
+      index: 'logs-endpoint.diagnostic.collection-default*',
```

Comment:
I think we need `logs-endpoint.diagnostic.collection-*` here, because I think @ferullo  was saying that the diagnostic alerts will respect the namespace setting, so they might come with something else than `default`.

Reply:
Thanks! Updated here: 2018132